### PR TITLE
feat: added avatar component

### PR DESCRIPTION
**Pull request by mansi2011**
Please review my code !

### CODE REVIEW

File: Components/Avatar/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.avatar-type{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--landing-hover-color);
+}
+
+iframe{
```
Comment (SanjayJatti) [inline Components/Avatar/Avatar.css:27]: For responsiveness, you can add max-width: 100% for carbon container

File: Components/Avatar/Avatar.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    padding: 1rem 2rem;
+    width: 110rem;
+    height: 22rem;
+    overflow-y: scroll;
+}
+
+.carbon-container{
```
Comment (SanjayJatti) [inline Components/Avatar/Avatar.css:34]: You can create flex-center utility class. As It may need you further also.

**Review by SanjayJatti — APPROVED** [review]
All code is looking clean and readable. Try to add live demo of component. Great work!

File: Components/Avatar/Avatar.css (hunk 2 above)
Comment (mansi2011) [reply Components/Avatar/Avatar.css:34]: okay will look into it.

Comment (mansi2011) [reply Components/Avatar/Avatar.css:34]: added the change

File: Components/Avatar/Avatar.css (hunk 1 above)
Comment (mansi2011) [reply Components/Avatar/Avatar.css:27]: added the change

**Comment by mansi2011** [comment]
avatar

Feat: added Avatar component
- small Avatar
- medium avatar
- large avatar

File: Components/Avatar/Avatar.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
+.small-avatar{
+    height: 7rem;
+    width: 7rem;
+    border-radius: 10rem;
```
Comment (Prabhav-Sharma) [inline Components/Avatar/Avatar.css]: You can define a global variable in the style guide file for border-radius

File: Components/Avatar/Avatar.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    width: 14rem;
+    border-radius: 10rem;
+}
+
+.avatar-demo{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 3rem 4rem;
```
Comment (Prabhav-Sharma) [inline Components/Avatar/Avatar.css:52]: For consistency in design, you would want to add a global variable for padding as well, something like 
--space-small:1rem
--space-xlarge:3rem

**Review by Prabhav-Sharma — CHANGES_REQUESTED** [review]

File: Components/Avatar/Avatar.css (hunk 3 above)
Comment (mansi2011) [reply Components/Avatar/Avatar.css]: Done thanks!

File: Components/Avatar/Avatar.css (hunk 4 above)
Comment (mansi2011) [reply Components/Avatar/Avatar.css:52]: okay will that in consideration for further use.

**Pull request merged**